qiskit/algorithms/minimum_eigensolvers/adapt_vqe.py

```diff
@@ -208,6 +251,8 @@ def compute_minimum_eigenvalue(
         self.solver.ansatz = self._tmp_ansatz.initial_state
 
         prev_op_indices: list[int] = []
+        prev_raw_vqe_result: VQEResult | None = None
+        raw_vqe_result: VQEResult | None = None
```

Comment:
Would it be sufficient to just store the previous energy instead of the whole result object?

Reply:
No because we combine the `VQEResult` into the `AdaptVQEResult` further down and we want to make sure to get the latest data for that.